dynamic_programming/cut_rod.cpp

```diff
@@ -1,25 +1,103 @@
-/*Given a rod of length n inches and an array of prices that
-contains prices of all pieces of size smaller than n. Determine
-the maximum value obtainable by cutting up the rod and selling
-the pieces.*/
+/**
+ * @file cut_rod.cpp
+ * @brief Implementation of cutting a rod problem
```

Comment:
Provide a Wikipedia link if possible, in Markdown format.

Reply:
I was also finding the relevant Wikipedia page but didn't found anything. If found will update in the future.

Reply:
You can link to other reqources like:
From universities: 
[this](https://web.stanford.edu/class/archive/cs/cs161/cs161.1168/lecture12.pdf)
[this](https://www.radford.edu/~nokie/classes/360/dp-rod-cutting.html)

Other: [this](https://www.educative.io/edpresso/the-rod-cutting-problem)